server/agents/default_agent_test.py

```diff
@@ -12,14 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# pylint: disable=missing-module-docstring, missing-function-docstring, missing-class-docstring, line-too-long
```

Comment:
is this necessary?

Reply:
My thinking is to have zero pylint errors ... If there's a bunch in tests that we just ignore then we'll never see when real ones crop up?